memory-viz/src/types/pretty.d.ts

```diff
@@ -0,0 +1,24 @@
+declare module "pretty" {
```

Comment:
Please remove this file from the branch and instead commit changes to the `package.json`/`package-lock` files, adding `@types/pretty` as a dev dependency. If that's not working for you I'll take a closer look.